src/main/java/core/basesyntax/service/operations/PurchaseOperation.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (Storage.fruits.containsKey(transaction.getFruit())

```

Comment:
There is no handling for the case when the fruit is not found in the Storage. It would be better to throw an exception or return a specific value indicating that the fruit was not found.

Reply:
ok, done